src/components/PaymentLog.vue

```diff
@@ -876,6 +876,11 @@ export default {
       try {
         const fetchSalesCategory = await get(this.apiUrls.salesCategoryUrl);
         this.salesCategories = fetchSalesCategory.data.data.data;
+        if(!this.isAuthorized){
+          this.salesCategories = this.salesCategories.filter(item => {
+            return item.name !== 'renewal(DSA)'
```

Comment:
why are we removing this from sales cateory? We only want to hide the discount field if you don't have access